### PR TITLE
Add OVA provider server image

### PR DESCRIPTION
**Pull request by bkhizgiy**
Add an OVA provider server that will create for each new ova provider in the system.
It will be in charge of filling the inventory by parsing the OVA files provided by the user.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -290,7 +299,8 @@ push-all-images:  push-api-image \
                   push-operator-index-image \
                   push-populator-controller-image \
                   push-ovirt-populator-image \
-                  push-openstack-populator-image
+                  push-openstack-populator-image\
+				  push-ova-provider-server-image
```
Comment (bkhizgiy) [inline Makefile:303]: I'm not sure why it's not aligned here, in my IDE everything is in place.

Comment (nyoxi) [reply Makefile:303]: The lines above are indented with spaces only. Your line uses mix of tabs and spaces and your IDE seems to be configured to show tab as 4 spaces (the common default is 8 spaces) that's why it aligns when you look at it.

File: hack/ova-provider-server/Containerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Builder stage
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18.9-8 as builder
+ENV GOPATH=$APP_ROOT
+COPY . .
+RUN CGO_ENABLED=0 go build -o ova-provider-server github.com/konveyor/forklift-controller/cmd/ova-provider-server
+
+# Final stage
+FROM quay.io/centos/centos:stream9-minimal
+COPY --from=builder /opt/app-root/src/ova-provider-server /usr/local/bin/ova-provider-server
+RUN microdnf install -y tar && microdnf clean all
```
Comment (ahadas) [inline hack/ova-provider-server/Containerfile:10]: we don't need `tar` now, right?

File: hack/ova-provider-server/Containerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Builder stage
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18.9-8 as builder
```
Comment (ahadas) [inline hack/ova-provider-server/Containerfile:2]: we can start using the one from ubi9

File: Makefile [hunk 4]
```diff
@@ -268,6 +270,12 @@ build-openstack-populator-image: check_container_runtmime
 push-openstack-populator-image: build-openstack-populator-image
 	$(CONTAINER_CMD) push $(OPENSTACK_POPULATOR_IMAGE)
 
+build-ova-provider-server-image: build-ova-provider-server-image
```
Comment (ahadas) [inline Makefile:273]:
```suggestion
build-ova-provider-server-image:  check_container_runtmime
```

File: Makefile [hunk 5]
```diff
@@ -268,6 +270,12 @@ build-openstack-populator-image: check_container_runtmime
...
+	$(CONTAINER_CMD) build -f hack/ova-provider-server/Containerfile -t $(OVA_PROVIDER_SERVER_IMAGE) .
+
+push-ova-provider-server-image: push-ova-provider-server-image
```
Comment (ahadas) [inline Makefile:276]:
```suggestion
push-ova-provider-server-image: build-ova-provider-server-image
```

File: Makefile (hunk 1 above)
Comment (bkhizgiy) [reply Makefile]: it's worked, Thanks!

**Review by ahadas — APPROVED** [review]

**Pull request merged**